Library/Formula/sslscan.rb

```diff
@@ -18,6 +18,7 @@ class Sslscan < Formula
   def install
     system "make"
     bin.install "sslscan"
+    man1.install "sslscan.1"
```

Comment:
It seems like we could use `system "make", "install", "BINPATH=#{bin}", "MANPATH="#{man}/"`:
https://github.com/rbsec/sslscan/blob/d0fdf8c38389d9d58b07e50ddd8f479e09a9b5d7/Makefile#L54